bitnami/contour/templates/contour/deployment.yaml

```diff
@@ -103,7 +103,8 @@ spec:
             - --contour-cert-file=/certs/tls.crt
             - --contour-key-file=/certs/tls.key
             - --config-path=/config/contour.yaml
-            - --envoy-service-name={{ printf "%s-envoy" (include "common.names.fullname" .) | trunc 63 | trimSuffix "-" }}
+            - --envoy-service-namespace={{ default .Release.Namespace .Values.contour.envoyServiceNamespace }}
+            - --envoy-service-name={{ default (printf "%s-envoy" (include "common.names.fullname" .) | trunc 63 | trimSuffix "-") .Values.contour.envoyServiceName }}
```

Comment:
Taking a deeper look at these new parameters you added, I noticed that the chart is using different values. You need to use the same ones we are already configuring in the envoy service.yaml file 

https://github.com/bitnami/charts/blob/main/bitnami/contour/templates/envoy/service.yaml#L5

Those new parameters you added are not needed and the version should be moved back to 10.1.6

Reply:
It is not necessary to specify a Service created in the envoy service.yaml file. In practice, users may create their own LoadBalancer Service or create multiple LoadBalancer Services. If only Services created in the envoy service.yaml file can be specified, users would have to make significant changes to their configuration. This PR is to avoid this issue and allow users to customize as they wish, as this feature is truly necessary for them.